BuildAndTest.proj

```diff
@@ -75,13 +76,17 @@
 
       <!-- MakeConst unit tests tracked by https://github.com/dotnet/roslyn/issues/5918 -->
       <TestAssemblies Condition="'$(Test64)' != 'true'" 
-                      Include="$(OutputDirectory)\**\$(IncludePattern)" />
+                      Include="$(OutputDirectory)\$(IncludePattern)" />
```

Comment:
Actually that might be the best way - then we'll never merge this into future without a build break.